### PR TITLE
Check if the Python code is correct

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

File: src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+private fun PsiElement.countPsiErrorElements(): Int {
+    return ApplicationManager.getApplication().runReadAction<Int> {
+        val elements = PsiTreeUtil.collectElements(this) { it is PsiErrorElement }
+        elements.size
+    }
+}
+
+// Find unresolved references errors
+fun PsiElement.canBeResolved(): Boolean {
+    this as @Nullable PsiFile
```
Comment (elena-lyulina) [inline src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt:32]: Not sure, what is this line for?

File: src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.research.ml.ast.util
```
Comment (elena-lyulina) [inline src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt:1]: Let's rename this file to reflect it's about PSI correctness, not just PSI

File: src/test/kotlin/org/jetbrains/research/ml/ast/util/FileTestUtil.kt [hunk 3]
```diff
@@ -57,4 +57,15 @@ object FileTestUtil {
         return folders.sortedBy { it.name }.map { getInAndOutFilesMap(it.absolutePath, inFormat, outFormat) }
             .fold(inAndOutFilesMap, { a, e -> a.plus(e) })
     }
+
+    // TODO: can we move some code into a common function?
```
Comment (elena-lyulina) [inline src/test/kotlin/org/jetbrains/research/ml/ast/util/FileTestUtil.kt:61]: I think we can

**Review by elena-lyulina — CHANGES_REQUESTED** [review]
Done great, good to see so many test cases, thanks :)

File: src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt (hunk 1 above)
Comment (nbirillo) [reply src/main/kotlin/org/jetbrains/research/ml/ast/util/Psi.kt:32]: I did not notice it, sorry. It is an old code line

**Review by elena-lyulina — APPROVED** [review]
Cool, thanks!

**Pull request merged**